src/app/home-page/widget/organization-box/organization-box.component.html

```diff
@@ -42,22 +42,31 @@
     <mat-divider></mat-divider>
     <mat-card-content class="p-3 h-100">
       <div *ngIf="totalOrgs !== 0 || isLoading; else noOrganizations">
-        <h5 class="weight-bold">Recent Activity</h5>
+        <mat-form-field appearance="outline" class="search-form-field w-100 pb-0">
+          <input
+            data-cy="filterOrganizationsInput"
+            matInput
+            [(ngModel)]="filterText"
+            (keyup)="onTextChange($event)"
+            placeholder="Find your organizations..."
+          />
+        </mat-form-field>
+        <h5 class="weight-bold mt-0">My Organizations</h5>
         <div
           *ngFor="let org of listOfOrganizations"
           fxLayout="row"
           fxLayoutAlign="space-between center"
           class="widget-list-items w-100 py-2"
         >
           <a class="no-wrap" [routerLink]="'/organizations/' + org.name">
-            {{ org.name }}
+            {{ org.displayName }}
           </a>
           <div class="date-display">{{ org.lastUpdateDate | date: 'MMM d, yyyy' }}</div>
         </div>
       </div>
       <ng-template #noOrganizations>
         <div *ngIf="!isLoading" fxLayout="column" class="h-100" fxLayoutAlign="center center">
-          <img src="../assets/svg/sub-nav/organization.svg" alt="my organizations" class="widget-box-type-logo mb-4" />
+          <img src="../../../../assets/svg/organization-circle.svg" alt="my organizations" class="widget-box-type-logo mb-4" />
           <span>You are not in any organizations </span>
           <div class="size-small text-align-center">
             <span>Use the </span>
```

Comment:
For line 67/76, I can't comment there, and it was already like this, but my IDE is flagging the id `requestOrganizationButton` as a duplicate id. Looks like the id isn't referenced anywhere, so I think you can just remove both ids.